### ISSUE TITLE
[ADD]: Addition of CI/CD 

### ISSUE THREAD

**Issue by AbhijithGanesh** [dokc/rebuilding-explorers-world#3]
# CI/CD for application

To make deployments and integrate changes to the repository easy, we usually have GitHub Actions to help us automate and track the changes. You can find the reference Action here, for this issue we would like the contributor to add a workflow which deploys a gatsby-application to a branch names `gh-pages`

**Comment by Islam-ansari** [comment]
hello there, I am learning about the DevOps (though im a beginner), but i'd like to contribute to this issue can you please guide me how to do it, and suggest me some resources etc.

**Comment by coldn00dles** [comment]
I would like to contribute

**Comment by akshat5302** [comment]
Hey, @AbhijithGanesh is this issue still open for contribution?

**Comment by AbhijithGanesh** [comment]
Sorry @akshat5302 ! I have assigned this to @Islam-ansari . But I would like to know if you'd like to share this @Islam-ansari

**Comment by Islam-ansari** [comment]
Yeah definitely, it's will be helpful I'm still a beginner, so more PPL involved less challenging it becomes

**Comment by akshat5302** [comment]
Ok

**Comment by akshat5302** [comment]
Could you just elaborate it more it would be helpful
